Thumbnail/ConsumerThumbnail.php

```diff
@@ -73,27 +73,27 @@ public function generatePrivateUrl(MediaProviderInterface $provider, MediaInterf
      */
     public function generate(MediaProviderInterface $provider, MediaInterface $media)
     {
-        $backend = $this->backend;
-        $id = $this->id;
+            $backend = $this->backend;
+            $id = $this->id;
 
-        $publish = function () use ($backend, $media, $id) {
-            $backend->createAndPublish('sonata.media.create_thumbnail', array(
-                'thumbnailId' => $id,
-                'mediaId' => $media->getId(),
+            $publish = function () use ($backend, $media, $id) {
+                $backend->createAndPublish('sonata.media.create_thumbnail', array(
+                    'thumbnailId' => $id,
+                    'mediaId' => $media->getId(),
 
-                // force this value as the message is sent inside a transaction,
-                // so we have a race condition
-                'providerReference' => $media->getProviderReference(),
-            ));
-        };
+                    // force this value as the message is sent inside a transaction,
+                    // so we have a race condition
+                    'providerReference' => $media->getProviderReference(),
+                ));
+            };
 
-        // BC compatibility for missing EventDispatcher
-        if (null === $this->dispatcher) {
-            trigger_error('Since version 2.3.3, passing an empty parameter in argument 4 for __construct() in '.__CLASS__.' is deprecated and the workaround for it will be removed in 3.0.', E_USER_DEPRECATED);
-            $publish();
-        } else {
-            $this->dispatcher->addListener('kernel.finish_request', $publish);
-        }
+            // BC compatibility for missing EventDispatcher
+            if (null === $this->dispatcher) {
+                trigger_error('Since version 2.3.3, passing an empty parameter in argument 4 for __construct() in '.__CLASS__.' is deprecated and the workaround for it will be removed in 3.0.', E_USER_DEPRECATED);
+                $publish();
+            } else {
+                $this->dispatcher->addListener('kernel.finish_request', $publish);
+            }
```

Comment:
No problem, we do use it sometimes but we prefer when contributor provide something as clean as possible from the start, it's easier to read.